cloud/proto/buf.gen.yaml

```diff
@@ -1,3 +1,14 @@
+
+ #                          _       _
+ #__      _____  __ ___   ___  __ _| |_ ___
+ #\ \ /\ / / _ \/ _` \ \ / / |/ _` | __/ _ \
+ # \ V  V /  __/ (_| |\ V /| | (_| | ||  __/
+ #  \_/\_/ \___|\__,_| \_/ |_|\__,_|\__\___|
+ #
+ # Copyright © 2016 - 2023 Weaviate B.V. All rights reserved.
+ #
+ # CONTACT: hello@weaviate.io
+ #
```

Comment:
Are those Weaviate headers autogenerated also for `yaml` files?

Reply:
I'm almost sure they were explicitely removed previously because they were managed with buf. Maybe we need to update an automation ?

Reply:
I've manually added this header. Auto-generation currently only works for Go files. We can consider implementing auto-generation later if necessary